pkg/sdk/go/sdk.go

```diff
@@ -347,28 +302,40 @@ func NewSDK(conf Config) SDK {
 	}
 }
 
-func (sdk mfSDK) sendRequest(req *http.Request, token, contentType string) (*http.Response, error) {
-	if token != "" {
-		req.Header.Set("Authorization", apiutil.BearerPrefix+token)
+func (sdk mfSDK) processRequest(method, url string, data []byte, token, contentType string, expectedResponseCode ...int) ([]byte, map[string][]string, errors.SDKError) {
```

Comment:
Change signature to:
```go
func (sdk mfSDK) processRequest(method, url, token, contentType string, data []byte, expectedRespCodes ...int) (http.Header, []byte, errors.SDKError) {...}
```
While it would be good not to expose any `HTTP` details, it's cleaner to return headers directly. Also, a method that accepts 6 parameters and returns 3 results doesn't sound like a good design, but, since it's a pretty simple implementation, we can accept it.

Reply:
Add a short comment explaining what this method does since its signature might look a little intimidating.